server/app.js

```diff
@@ -11,6 +11,18 @@ require( 'dotenv' ).load();
 var Logger = require( './components/Logger' );
 Logger.initialize();
 
+if( true ) {
+  if( true ) {
+    console.log( 'CACA' );  console.log( 'CULO' ); 
+  }
+}
+
+if( true ) {
+  if( true ) {
+    console.log( 'CACA' );  console.log( 'CULO' ); 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube:9000/coding_rules#rule_key=javascript%3AOneStatementPerLine)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube:9000/coding_rules#rule_key=javascript%3AS2228)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube:9000/coding_rules#rule_key=javascript%3AS2228)